tests/e2e/ADDING_TESTS.md

```diff
@@ -68,7 +68,11 @@ const test = () => {
   Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(firstReportIDInList));
 
   // markEnd will be called in the Screen's implementation
-  performance.subscribeToMeasurements("navigateToReport", (measurement) => {
+  Performance.subscribeToMeasurements((measurement) => {
+      if (!measurement.name !== "navigateToReport") {
```

Comment:
What is this early return here for? Also, why is there a "!" before `measurement.name`?

Reply:
cc @hannojg but I see the e2e tests are just designed this way - we listen to all events and only want to proceed further on specific ones - in this case `navigateToReport`.

Reply:
ah the `!` is my typo, thanks!